src/pixelator/plot/__init__.py

```diff
@@ -47,6 +57,56 @@ def _calculate_densities(coordinates, distance_cutoff, unit_sphere_surface):
     return densities
 
 
+def pxContent_vs_Tau(data: pd.DataFrame, group_by: Optional[str] = None) -> ggplot:
+    """Create a scatter plot of pixel content vs marker specificity (Tau).
+
+    :param data: a pandas DataFrame with the columns 'umi_per_upia', 'tau', and 'tau_type'.
+    :param group_by: a column in the DataFrame to group the plot by.
+
+    :return: a scatter plot of pixel content vs marker specificity (Tau).
+    :rtype: ggplot
+    :raises: ValueError if the required columns are not present in the DataFrame
+    :raises: AssertionError if the data types are invalid
+    """
+    # Validate data
+    required_columns = ["umi_per_upia", "tau", "tau_type"]
+    if not all(col in data.columns for col in required_columns):
+        raise ValueError(
+            "'umi_per_upia', 'tau' and 'tau_type' must be available in the DataFrame"
+        )
+
+    assert all(
+        [
+            data["umi_per_upia"].dtype == "float64",
+            data["tau"].dtype == "float64",
+            data["tau_type"].dtype in ["object", "category"],
+        ]
+    ), "Invalid data types"
+
+    if group_by is not None:
+        if group_by not in data.columns:
+            raise ValueError(f"'{group_by}' is missing")
+        assert data[group_by].dtype in [
+            "object",
+            "category",
+        ], "'group_by' must be a character or factor"
+
+    # Create plot
+    plot = (
+        ggplot(data, aes(x="tau", y="umi_per_upia", color="tau_type"))
+        + geom_point()
+        + (facet_grid("~" + group_by) if group_by is not None else "")
+        + scale_y_log10()
+        + scale_color_manual(
+            values={"high": "#ee4000", "low": "#6ca6cd", "normal": "#bebebe"}
```

Comment:
Can we encode these color codes in e.g. a constants module?

Reply:
What does that mean? :-/

Reply:
There's now a constants module where the colors are defined.